tools/unicode_properties_parse/grapheme_break_property_data_gen.py

```diff
@@ -34,6 +34,26 @@ def parsePropertyLine(inputLine: str) -> Optional[PropertyRange]:
     else:
         return None
 
+def compactPropertyRange(input: list[PropertyRange]) -> list[PropertyRange]:

```

Comment:
Naming: I think this should be `compactPropertyRanges` because it compacts a sequence of ranges, not an individual one.